README.md

```diff
@@ -226,4 +226,27 @@ Example body: ( To add a doctor change the `profession` value to `doctor` and om
     insurance: ["BigMoney", "OtherBigMoney"],
     specialties: ["Stuff", "Things", "Problems"]
 }
+```
+
+<hr>
+
+#### PATCH `/medical_professionals/#id` </br> 
+A successful update request will return 204 No Content </br> 
+A request missing required params will return 422 Unprocessable Entity </br> 
+A request missing an api key will return 401 Unauthorized </br> 
+
+headers:
+```
+"api-key" : "your_api_key",
+"Content-Type": "application/json"`, 
+"Accept": "application/json"
+```
+Required params: "id", "first_name", and "last_name"
```

Comment:
I think we only need to require the id?

Reply:
> o

Oh, this is for the Approve button on the admin dashboard! Sorry for the ambiguity.